core/src/main/java/nl/nn/adapterframework/scheduler/job/CheckReloadJob.java

```diff
@@ -36,7 +36,22 @@
 
 public class CheckReloadJob extends JobDef {
 	private static final boolean CONFIG_AUTO_DB_CLASSLOADER = AppConstants.getInstance().getBoolean("configurations.database.autoLoad", false);
+	private static final String DATABASE_CLASSLOADER = "DatabaseClassLoader";
+	private boolean atLeastOneConfigrationHasDBClassLoader = CONFIG_AUTO_DB_CLASSLOADER;
 
+	@Override
+	public boolean isRunJob(IbisManager ibisManager) {
```

Comment:
Perhaps instead of creating a new method you can reuse `configure` and `beforeExecuteJob`.
Either move this to `configure`, (but then it will only be called once)
Or make `beforeExecuteJob` return a boolean to determine if the job should be triggered.